components/SidePillTag.vue

```
@@ -0,0 +1,215 @@
+  height: 56,
+  openWidth: 320,
+  peekWidth: 120,
+
+  lineColor: '#E54B4B',
+  lineWidth: 2,
+  radius: 28,
+
+  useElementPlus: false,
+})
+
+const open = ref(false)
```

ESLint

[error] Insert `;`

(prettier/prettier)